### ISSUE TITLE
Message Notifications only show if map is open

### ISSUE THREAD

**Issue by rrgeorge** [encounterplus/encounterplus#530]
### What went wrong?
- New message notifications only show if a map is open

### What were you expecting to happen?
- To receive message notifications if a map is not open

### What steps can we follow to reproduce the issue?
1. Open E+
2. Make sure there are not maps open
3. Roll a die or send a message from the webclient
4. Observe there’s no notifications in E+

### What software versions are you currently using?
- EncounterPlus (e.g. 4.11.1): 4.11.2
- Operating System (e.g. "iOS/iPadOS 14.2" or "macOS 11.3"): 14.6

### Any other comments?
-

**Comment by jurex** [comment]
yeah, the message stack UI is part of the map controller. Need to move it to the main screen, so it can be visible regardless of map state.
